Iih update

PranglB opened the pull request:
Update IIH related stuff and KPI Type definition

PranglB committed:
Modified images and Readme to Common Configurator
and IIH Essentials

PranglB committed:
inserted images

PranglB committed:
changed images

PranglB committed:
inserted back KPI Instance

HummelN7872 commented on docs/Installation.md:
```diff
@@ -8,26 +8,33 @@
 - Data Service
 
-Please refer to [using the Data Service](https://github.com/industrial-edge/data-service) for detailed instructions.
+Please refer to [using the IIH](https://github.com/industrial-edge/data-service) for detailed instructions.
```
Could you please adjust this link to: https://github.com/industrial-edge/iih-essentials-getting-started 
The repos is reachable via both URLs, but data-service is the legacy name,

HummelN7872 commented on docs/Installation.md:
```diff
@@ -8,26 +8,33 @@
 - Databus
 - OPC UA Connector
+- Common Configurator
 - Data Service
```
Please rename in IIH

PranglB committed:
changed data service to IIH stuff

HummelN7872 commented on docs/graphics/OPCUA_Connector_Configuration.PNG:
it is neither possible in the common configurator nor in the iih configuration to see, which variables are used in this example. Could you please drag the colums bigger, that the variable names are readable , there is enough white space availble. You could even think about crop some of the white parts out, to make it better readable

HummelN7872 commented on docs/graphics/Data_Service_Data_Service_Variable.PNG:
it is neither possible in the common configurator nor in the iih configuration to see, which variables are used in this example. Could you please drag the colums bigger, that the variable names are readable , there is enough white space availble. You could even think about crop some of the white parts out, to make it better readable

HummelN7872 replied:
@PranglB  Could you please do the same with the screenshot of the Common Configurator:
<img width="1030" height="337" alt="image" src="https://github.com/user-attachments/assets/dcf9b273-7d6f-416c-b883-51346c71012c" />

Once done I am happy and we can merge it

PranglB committed:
changed IIH Essentials pic to show full Tagname
Insert mapping for energy app and hint to change display names of tags
Energy App only shows full Tagname when hovering over the tag

PranglB committed:
get common configurator config readable
